src/commands/configure.js

```diff
@@ -22,6 +22,11 @@ class ConfigureCommand extends TerraformCommand {
    */
   run() {
     const configContent = this.getOption('config');
+
+    if (!configContent) {
```

Comment:
@Leichtwind please help me here; i thought we have a centralized functionality to check for missing options